### PR TITLE
Adds aggregation/regular path KatanaGraph queries, flatten query directories

### PR DESCRIPTION
**Pull request by l-hoang**
Adds the queries created to test preliminiary aggregation support as
well as the revival of the regular path queries.

They do not all run at the time this commit message was written. For
this reason, they are not added to the test suite for CI just yet.

Also flattens the query directory structure.

### CODE REVIEW

**Comment by l-hoang** [comment]
also note that some of the queries return no match for ldbc_003; to get non-zero matches you have to run on at least scale01.

File: lonestar/querying/cpu/graph-query/cypher-queries/003_phase1_queries.txt.truth [hunk 1]
```diff
@@ -0,0 +1,7 @@
+NOTE: this is number of rows: better to use this to check if aggregation
+gets you the correct # of rows
+count of phase1-1.q is 0
+phase1-2.q 17
```
Comment (ddn0) [inline lonestar/querying/cpu/graph-query/cypher-queries/003_phase1_queries.txt.truth:4]: phase1 is an arbitrary name that is likely to go stale very quickly. Either give them a name that reflects their content or just give them an arbitrary number (without phase1).

Comment (l-hoang) [reply lonestar/querying/cpu/graph-query/cypher-queries/003_phase1_queries.txt.truth]: changed the names, let me know what you think

File: lonestar/querying/cpu/graph-query/cypher-queries/003_phase1_queries.txt.truth [hunk 2]
```diff
@@ -0,0 +1,7 @@
+NOTE: this is number of rows: better to use this to check if aggregation
```
Comment (roshandathathri) [inline lonestar/querying/cpu/graph-query/cypher-queries/003_phase1_queries.txt.truth]: Don't use phase1 for this file name as well. Just say "untested_queries"

File: lonestar/querying/cpu/graph-query/cypher-queries/phase1_queries.txt [hunk 3]
```diff
@@ -0,0 +1,5 @@
+/katana/likes_own_messages.q
```
Comment (roshandathathri) [inline lonestar/querying/cpu/graph-query/cypher-queries/phase1_queries.txt]: Ditto.

File: lonestar/querying/cpu/graph-query/cypher-queries/unsupported.txt [hunk 4]
```diff
@@ -0,0 +1,5 @@
+/katana/likes_own_messages.q
```
Comment (ddn0) [inline lonestar/querying/cpu/graph-query/cypher-queries/unsupported.txt:1]: I see in `lonestar/querying/cpu/graph-query/cypher-queries/` subdirectories like:

- katana
- single_edge
- single_edge_w_nl
- single_node
- three
- two

If I was looking to add a new query (test), where would I put it? If it has two nodes and no aggregation, is that "two"? If it has (non-trivial) aggregation, "katana"?

If there isn't an obvious (and unique) place, these subdirectories will quickly become meaningless and we might as well just get rid of them.

Comment (l-hoang) [reply lonestar/querying/cpu/graph-query/cypher-queries/unsupported.txt:1]: done

File: lonestar/querying/cpu/graph-query/cypher-queries/003_unsupported.txt.truth [hunk 5]
```diff
@@ -0,0 +1,8 @@
+NOTE: this is number of rows: better to use this to check if aggregation
+gets you the correct # of rows
+
+count of likes_own_messages.q is 0
+people_work_in_school_country.q 17
+person_like_tag_not_interested.q 8
+count of friend_of_friend_messages.q  is 0
```
Comment (roshandathathri) [inline lonestar/querying/cpu/graph-query/cypher-queries/003_unsupported.txt.truth:7]: Drop "count of". Please sort the list by name.

File: lonestar/querying/cpu/graph-query/cypher-queries/querylist.txt [hunk 6]
```diff
@@ -1,50 +1,50 @@
...
-/katana/demo1.q
-/katana/demo2.q
-/katana/demo3.q
+containerof.q
+hascreator.q
+hasinterest.q
+hasmember.q
+hasmoderator.q
+hastag.q
+hastype.q
```
Comment (roshandathathri) [inline lonestar/querying/cpu/graph-query/cypher-queries/querylist.txt:7]: Please sort this list by name.

File: lonestar/querying/cpu/graph-query/cypher-queries/unsupported.txt [hunk 7]
```diff
@@ -0,0 +1,5 @@
...
+/katana/people_work_in_school_country.q
+/katana/person_like_tag_not_interested.q
```
Comment (roshandathathri) [inline lonestar/querying/cpu/graph-query/cypher-queries/unsupported.txt:3]: Ditto.

**Pull request merged**